packages/block/src/block.ts

```diff
@@ -96,12 +99,21 @@ export class Block {
     // parse transactions
     const transactions = []
     for (const txData of txsData ?? []) {
-      const tx = TransactionFactory.fromTxData(txData, {
-        ...opts,
-        // Use header common in case of hardforkByBlockNumber being activated
-        common: header._common,
-      } as TxOptions)
-      transactions.push(tx)
+      let skip = false
+      if (opts?.skipTxTypes !== undefined && txData.type !== undefined) {
+        const txType = Number(bytesToInt(toBytes(txData.type)))
+        if (opts.skipTxTypes.includes(txType)) {
+          skip = true
+        }
+      }
+      if (!skip) {
```

Comment:
Is it possible to include the system transaction in the block's transaction list and just not validate that specific transaction or are there further complications if you include a system transaction in a block unvalidated?